docs/rapid-online/4. RapidPlan Online Workspace/Palettes.md

```diff
@@ -5,24 +5,22 @@ sidebar_position: 8
 ---
 # Palettes
 
-The palettes are located on the left (Tools) and right (Operations) of the canvas area. They can be shown/hidden by toggling corresponding button on the Toolbar. This can be helpful for with maximizing the canvas area when preparing larger scale plans.
+The Palettes are located on the left and right of the Canvas Area. Whole toolbars or specific Palettes can be shown/hidden by toggling the corresponding arrow button as pictured.
 
-The Tools palette includes:
+On the left side you'll find Palettes that includes everything that can be placed, imported and designed on your plan:
 
-* Tools
+- [**The Tools Palette**](./Tools%20palette.md)
+- [**Scratchpad:**](./Scratchpad%20palette.md)
+- [**Signs:**](./Signs%20palette.md)
 
-* Scratchpad; and
+On the right, Palettes alter an object or your plan:
 
-* Signs.
-
-The Operations palette includes:
-
-* Properties
-
-* History; and
-
-* Layers.
-
-Each palette can be minimized by clicking the arrow icon at the top left.
+- [**Properties**](./Properties%20palette.md)
```

Comment:
it's better to have name like ``properties-palette.md`` with kebab case all small latters